src/components/AvatarList.astro

```diff
@@ -1,10 +1,48 @@
+---
+export let path;
+// fetch all commits for just this page's path
+let url = `https://api.github.com/repos/snowpackjs/astro-docs/commits?path=${path}`;
+let data = await fetch(url, {
+  method: "GET",
+  headers: {
```

Comment:
> I'm getting a 403 Forbidden through my personal token, but I can load the URL fine in Firefox... so A. It's probably a good idea for me to figure out how to use Octokit and B. auth is confusing

Thanks for tackling this! 

I believe you're getting a 403 because you don't have a User-Agent declared in the request headers. https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent#library_and_net_tool_ua_strings

Reply:
Done. It loads correctly for me now, but that doesn't necessarily mean that this perfectly fixed everything. I remember the API docs asking to set a User-Agent, so thanks for the recommendation!